signer/fourbyte/fourbyte.go

```diff
@@ -111,7 +111,7 @@ func (db *Database) Selector(id []byte) (string, error) {
 	if selector, exists := db.custom[sig]; exists {
 		return selector, nil
 	}
-	return "", fmt.Errorf("signature %v not found", sig)
+	return "", fmt.Errorf("signature %v not found, consider using --advanced mode or provide a selector", sig)
```

Comment:
This message will appear regardless of whether `--advanced` is used or not, and it's a bit wonky if the app asks you to apply a flag you already apply. 

However, on a higher level --  I think putting this info into this particular error message is not right: the fourbyte package should not make assumptions about the command line parameters of the app that invokes it.

Reply:
make sense, but it seems like that a lot of people have the confused if they don't start with -- advanced mode. should we highlight it in the doc, or something else？

Reply:
What I'm saying is that if you want to put an informative message, a better location is here: 
```golang
	// If we are in 'rejectMode', then reject rather than show the user warnings
	if api.rejectMode {
		if err := msgs.GetWarnings(); err != nil {
			return nil, err
		}
	}
````
in `signer/core/api.go`

Reply:
> What I'm saying is that if you want to put an informative message, a better location is here:
> 
> ```go
> 	// If we are in 'rejectMode', then reject rather than show the user warnings
> 	if api.rejectMode {
> 		if err := msgs.GetWarnings(); err != nil {
> 			return nil, err
> 		}
> 	}
> ```
> 
> in `signer/core/api.go`

done, plz check again